networkx/algorithms/bipartite/extendability.py

```diff
@@ -0,0 +1,141 @@
+""" Provides a function for computing the extendability of a
+graph which is undirected, simple, connected and bipartite. """
+
+
+import networkx as nx
+from networkx.utils import not_implemented_for
+
+
+__all__ = [
+    "find_extendability",
+]
+
+
+@not_implemented_for("directed")
+@not_implemented_for("multigraph")
+def find_extendability(G):
+    """Computes the extendability of a graph.
+
+    Definition
+    Graph G is $k$-extendable if and only if G has a perfect matching and every
+    set of $k$ independent edges can extend to perfect matching.
+
+    EXTENDABILITY PROBLEM
+    Input: A graph G and a positive integer $k$.
+    Output: Is G $k$-extendable?
+
+    In general case the above problem is co-NP-complete([2]).
+    If graph G is bipartite, then it can be decided in polynomial time([1]).
+
+    The maximization version of the EXTENDABILITY PROBLEM asks to compute the
+    maximum $k$ for which G is $k$-extendable.
+
+    Let G be a simple, connected, undirected and bipartite graph with a perfect
+    matching M and bipartition (U,V). The residual graph of G, denoted by $G_M$,
+    is the graph obtained from G by directing the edges of M from V to U and the
+    edges that do not belong to M from U to V.
+
+    Lemma([1])
+    Let M be a perfect matching of G. G is k-extendable if and only if its residual
+    graph $G_M$ is strongly connected and there are $k$ vertex-disjoint directed
+    paths between every vertex of U and every vertex of V.
```

Comment:
Can you add a sentence or two after the Lemma to explain why/how this lemma relates to this function? Are you explaining the algorithm you use -- there must be a reason to include this lemma at this point. 

This lemma also leads to a question of whether you should provide a function that returns the residual graph.